### PR TITLE
11.0 mig server environment ir config parameter

### PR DESCRIPTION
**Pull request by TDu**

### CODE REVIEW

File: server_environment_ir_config_parameter/README.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+For example you can use this module in combination with web_environment_ribbon:
+
+.. code::
+
+   [ir.config_parameter]
+   ribbon.name=DEV
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/10.0
```
Comment (yajo) [inline server_environment_ir_config_parameter/README.rst]: 11.0

**Review by yajo — CHANGES_REQUESTED** [review]
Just on little detail.

File: server_environment_ir_config_parameter/README.rst (hunk 1 above)
Comment (TDu) [reply server_environment_ir_config_parameter/README.rst]: Done

**Comment by yajo** [comment]
Bots are ❌

**Comment by TDu** [comment]
About the red test, seems that the package python-mysqldb [in travis.yml](https://github.com/OCA/server-tools/blob/ea9b0d58fcadc7792a9b4f570b32a947c7d5ffcf/.travis.yml#L16) does not support Python 3

**Comment by yajo** [comment]
Then please take the chance to remove it.

**Comment by TDu** [comment]
Waiting for server_environment [to be migrated](https://github.com/OCA/server-tools/pull/1051)

**Review by grindtildeath — APPROVED** [review]
Tested with https://github.com/OCA/server-tools/pull/1051 and works well :+1:

**Comment by simahawk** [comment]
@TDu can you rebase and add the oca_dependencies.txt to include #1051 until is merged?

**Review by simahawk — CHANGES_REQUESTED** [review]
LGTM, just include #1051 to make travis happy :)

**Comment by TDu** [comment]
I added oca_dependencies.txt
But server_environment depends on server_environment_files. I changed .travis.yml to create it like on branch 10.0 but still not working...

**Comment by pedrobaeza** [comment]
Closing in favor of https://github.com/OCA/server-env/pull/8

**Pull request closed without merging**